src/builder/rollup.ts

```diff
@@ -179,24 +179,36 @@ export async function rollupBuild(ctx: BuildContext) {
     await ctx.hooks.callHook("rollup:dts:options", ctx, rollupOptions);
     const typesBuild = await rollup(rollupOptions);
     await ctx.hooks.callHook("rollup:dts:build", ctx, typesBuild);
+    if (ctx.options.rollup.emitCJS) {
+      await typesBuild.write({
```

Comment:
Can we parallelize it when both CJS and ESM are emitted?

Reply:
The above tasks are also executed sequentially.
For consistency, I also executed it sequentially here.